pkg/kubelet/deviceplugin/types.go

```diff
@@ -72,3 +72,8 @@ const (
 	// errListAndWatch is the error raised when ListAndWatch ended unsuccessfully
 	errListAndWatch = "listAndWatch ended unexpectedly for device plugin %s with error %v"
 )
+
+const (
```

Comment:
Can we move this const def to manager.go, probably right above CheckpointFile() that uses it.